argilla-server/src/argilla_server/_app.py

```diff
@@ -180,21 +180,35 @@ def configure_telemetry():
         _LOGGER.warning(message)
 
 
-async def configure_database():
-    async def check_default_user(db: AsyncSession):
-        def _user_has_default_credentials(user: User):
-            return user.api_key == DEFAULT_API_KEY or accounts.verify_password(DEFAULT_PASSWORD, user.password_hash)
-
-        default_user = await accounts.get_user_by_username(db, DEFAULT_USERNAME)
-        if default_user and _user_has_default_credentials(default_user):
-            _LOGGER.warning(
-                f"User {DEFAULT_USERNAME!r} with default credentials has been found in the database. "
-                "If you are using argilla in a production environment this can be a serious security problem. "
-                f"We recommend that you create a new admin user and then delete the default {DEFAULT_USERNAME!r} one."
-            )
+async def _create_allowed_workspaces(db: AsyncSession):
+    from argilla_server.security.settings import settings as security_settings
+
+    if not security_settings.oauth.enabled:
+        return
+
+    for allowed_workspace in security_settings.oauth.allowed_workspaces:
+        if await accounts.get_workspace_by_name(db, name=allowed_workspace.name) is None:
```

Comment:
```suggestion
        if await Workspace.get_by(db, name=allowed_workspace.name) is None:
```

And then you can remove `get_workspace_by_name` from `accounts` context module.